core/connection/manager_test.go

```diff
@@ -109,12 +109,16 @@ func (tc *testContext) SetupTest() {
 		},
 	}
 
-	tc.ipCheckParams = IPCheckParams{
-		MaxAttempts:             3,
-		SleepDurationAfterCheck: 1 * time.Millisecond,
-		Done:                    make(chan struct{}, 1),
+	tc.params = Params{
+		IPCheck: IPCheckParams{
+			MaxAttempts:             3,
+			SleepDurationAfterCheck: 1 * time.Millisecond,
+		},
+		KeepAlive: KeepAliveParams{
+			SendInterval:    100 * time.Millisecond,
```

Comment:
Too frequent.
Also we will disconnect session after 500ms of temporarily connectivity issues.

Reply:
This is unit test :)